opsdroid/events.py

```diff
@@ -451,6 +451,9 @@ def __init__(self, description, *args, **kwargs):  # noqa: D107
 class JoinRoom(Event):
     """Event class to represent a user joining a room."""
 
+    def __init__(self, *args, **kwargs):  # noqa: D107
```

Comment:
Do we need an `__init__` here if it's not doing anything?

Reply:
Was not sure if its need to call the `super.init`. As no additional parameters are needed.. what would be the correct constructor?

Reply:
If you omit the `__init__` it will call the super init so don't worry. You can remove this.